test/teams.js

```diff
@@ -0,0 +1,122 @@
+const assert = require('assert');
+const supertest = require('supertest');
+
+const app = require('../express/routes');
+const models = require('../sequelize/models');
+
+describe('GET /teams', () => {
+    it('should return all teams and have status code 200', () => {
+        supertest(app)
+            .get('/teams')
+            .expect(200);
+    });
+});
+
+describe('GET /teams/:id', () => {
+    it('should return team and have status code 200', async () => {
+        const testTeam = await models.team.findOne();
+
+        const res = supertest(app)
+            .get(`/teams/${testTeam.id}`)
+            .expect(200);
+    });
+});
+
+describe('POST /teams', () => {
+    it('should create team and have status code 201', async () => {
+        const newTestTeam = {
+            team: {
+                teamName: 'Lokomotiv'
+            }
+        };
+
+        const numberOfTeamsBefore = await models.team.count();
+
+        const res = await supertest(app)
+            .post('/teams')
+            .send(newTestTeam)
+            .expect(201);
+
+        const numberOfTeamsAfter = await models.team.count();
+        const teamById = await models.team.findByPk(res.body.team.id);
+
+        assert.strictEqual(numberOfTeamsBefore + 1, numberOfTeamsAfter, 'Should create team');
+        assert.strictEqual(newTestTeam.team.teamName, teamById.teamName, 'Should create correct team');
+    });
+
+    it('should return validation error and have status code 400', async () => {
+        const newTestTeam = {
+            team: {
+                teamName: 'C'
+            }
+        };
+
+        const res = await supertest(app)
+            .post('/teams')
+            .send(newTestTeam)
+            .expect(400);
+    });
+});
+
+describe('PUT /teams/:id', () => {
+    it('should update team and have status code 200', async () => {
+        const newTeamName = 'newTeamName';
+
+        const testTeamBefore = await models.team.findOne();
+        testTeamBefore.teamName = newTeamName;
+
+        const editTestTeam = { team: testTeamBefore.toJSON() };
+
+        const res = await supertest(app)
+            .put(`/teams/${testTeamBefore.id}`)
+            .send(editTestTeam)
+            .expect(200);
+
+        const testTeamAfter = await models.team.findByPk(testTeamBefore.id);
+
+        assert.strictEqual(testTeamAfter.teamName, newTeamName, 'Should update teamName');
+    });
+
+    it('should return validation error and have status code 400', async () => {
+        const newTeamName = 'Bayern';
+
+        const testTeamBefore = await models.team.findOne();
+        testTeamBefore.teamName = newTeamName;
+
+        const editTestTeam = { team: testTeamBefore.toJSON() };
```

Comment:
Do we need to get real team here to trigger validation error? Or we can just send any invalid json?